legal/security_research.md

```diff
@@ -0,0 +1,60 @@
+# Policies for Security/Community Research
+
+This document lays out the Julia projects policies and guidance for researchers
+wishing to perform research on Julia's software development processes,
+contributors or other aspects of the julia community,
+
+For security issues in the software itself, please see the [Security policy](https://github.com/JuliaLang/julia/security/policy)
+or email [security@julialang.org](security@julialang.org).
+
+## Scope
+
+This document applies to the Julia programming language itself, the Julia package registries, as well as any packages directly affiliated with the Julia project. *Additionally, wherever this policy imposes any notification, disclosure or other requirements, such requirements also apply for research involving any packages registered in the Julia package registry (and thus distributed by the Julia project)*.
+
+## Non-interventional research
+
+Passive analysis of public information is generally fine. However, please ensure that your retrieval of such information is not disruptive (e.g. no excessive numbers of request submission to community infrastructure, or excessive bandwidth usage). Additionally, when referring to specific actions or inactions by specific members of the community in published writing, please follow these best practices:
```

Comment:
> Passive analysis of public information is generally fine.

Could we clarify that it's up to the researchers to ensure compliance with relevant laws?

Public information attached to projects, in particular attached to git commits and such, can be personal data and processing and storing it may or may not come with legal requirements. I think it's important to outline that it's up to the researchers to ensure they conform to any such requirements (unfortunately many don't), and that the "approval" in this document does not act as any kind of opt-in in the name of contributors of the projects covered by the scope of this doc.